variable/rebin.cpp

```diff
@@ -108,6 +108,10 @@ Variable rebin(const VariableConstView &var, const Dim dim,
     throw except::BinEdgeError(
         "The input does not have coordinates with bin-edges.");
 
+  if (is_bins(var))
+    throw except::BucketError("The input variable cannot be of type bins. Use "
+                              "bin or histogram instead of rebin.");
```

Comment:
How do we want to refer to such variables? I think I have used "binned data" or "binned variables" in the docs, but I am not entirly happy about that?

Suggest to add quotes:

```suggestion
    throw except::BucketError("The input variable cannot be of type bins. Use "
                              "`bin` or `histogram` instead of `rebin`.");
```

Reply:
In the code comments `bin-variable` is used but I don't think that's a vast improvement. I'll open that question to the scipp channel.

Reply:
Also a good discussion item for this afternoon.